driver-mct/cime_config/config_component.xml

```diff
@@ -1898,40 +1898,58 @@
     <desc>glc2ocn runoff mapping file decomp type for ice runoff</desc>
   </entry>
 
-  <entry id="OCN2GLC_FMAPNAME">
```

Comment:
Since these entries are being removed, they also need to be removed/renamed everywhere in config_grids.xml.  I see a few "OCN2GLC_FMAPNAME" and "OCN2GLC_SMAPNAME" left.

Reply:
@rljacob , thanks for catching the unmodified OCN2GLC_*MAPNAME entries in config.grids.xml.  Those grids were recently added after I had started this PR, and I forgot to modify them after rebasing this branch recently.  I've pushed a new commit to take care of them.